.github/workflows/dev.yml

```diff
@@ -0,0 +1,38 @@
+name: dev build
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
Do we still need `master`?

```suggestion
```

Reply:
We still have some repos with `master` as its default branch.  We will discuss this in the next team meeting, but for now we probably have to keep both for consistency across repos.